src/uct/api/uct.h

```diff
@@ -290,7 +290,13 @@ enum {
      * If set, the component supports @ref uct_cm_h functionality.
      * See @ref uct_cm_open for details.
      */
-    UCT_COMPONENT_FLAG_CM = UCS_BIT(0)
+    UCT_COMPONENT_FLAG_CM       = UCS_BIT(0),
+
+    /**
+     * Component supports direct access to remote memory via a pointer that is
+     * returned by @ref uct_rkey_ptr
```

Comment:
`If set, the component supports direct access to remote memory using a local pointer returned from @ref uct_rkey_ptr function`